atlas-search/wp-engine-search.mdx

```diff
@@ -0,0 +1,299 @@
+
+![WPGraphQL Debug Setting](./images/as-wpgraphql-debug-settings.png)
+
+#### Query Monitoring for non-WPGraphQL Users
+
+WP Engine Search works seamlessly with the [Query Monitor plugin](https://wordpress.org/plugins/query-monitor/) for non-WPGraphQL users (see "Debug Information" above for how to access logs if you are using WPGraphQL).
+
+The query monitoring plugin will allow users to view detailed information about the Atlas Search queries being made on their site, including the query string, the time it took to execute the query, and more. This information can be used to optimize search performance and ensure that searches are working as expected.
+
+To use Query Monitor with WP Engine Search, follow these steps:
+
+1. Install the [Query Monitor plugin](https://wordpress.org/plugins/query-monitor/) on your WordPress site.
+2. Activate the plugin.
+3. Click the Query Monitor that has appear in your WordPress Admin Top Nav.
+4. In the dashboard that appears, click on "Logs."
+5. Run Atlas Search and see the data appear.
+
+### Querying with WP Engine Search
+
+Atlas Search works completely out of the box, no code changes needed. However, some advanced customers prefer to use [WPGraphQL](https://www.wpgraphql.com/docs/introduction) to write custom queries. If this describes you, or is interesting to you, read along. If not, don't worry! Atlas Search will do everything in the above documentation, with no code changes required!
+
+Once your site is set up with an WP Engine Search beta license and the initial content sync has completed, WP Engine Search will take over the default WordPress search functionality and enhance your WordPress search experience.
+
+<em>
+  **Note:** If you have other search tools installed on your site, you may need
+  to deactivate or uninstall them for WP Engine Search to work properly.
+</em>
+
+For advanced users wanting to test search queries, the WPGraphQL plugin is a good place to start. WPGraphQL provides a ‘GraphiQL IDE' in the WordPress dashboard where you can interact with the WPGraphQL Schema in the browser. WP Engine Search overrides the default search functionality for WPGraphQL queries on all WP Engine Search supported WordPress data types.
+
+Below is a screenshot of the WPGraphQL plugin & GraphiQL IDE view, where WP Engine Search has overridden the default WP search behavior, followed by some sample queries:
+
+![Atlas Search Plugin Data Sync](./images/as-wpgraphiql.png)
+
+#### Querying Standard WordPress Data Types
+
+Here are examples of querying standard WordPress data types (Posts, Pages, and Custom Post Types (CPT)):
+
+- Querying Posts/Pages:
+
+```graphql
+{
+  {either "posts" or "pages"}(where: {search: "CSS"}) {
+    nodes {
+      id
+      status
+      title
+    }
+  }
+}
+```
+
+- Querying Custom Post Types:
+
+```graphql
+{
+  branches(where: { search: "Austin" }) {
+    nodes {
+      address
+    }
+  }
+}
+```
+
```

Comment:
I think we can frame this in the view of the WordPress search bar instead of WP GraphQL
<img width="1522" alt="image" src="https://github.com/wpengine/headless-docs/assets/24898309/eff8780a-fd73-4431-b117-887e7a8fb3e4">